DataManager.h

```diff
@@ -104,6 +104,7 @@ class DataManager : public CBase_DataManager {
         // * either do not concern yourself with cached particles
         // * or for each entry, get key, find bucket node in CM, DM or TPs and get number
         // for now, the former
+	int treePiecesBufferFilled;
```

Comment:
This line has been inserted between a comment (above) and the variable it is describing (below).  This should be moved and also commented.